.ci/ansible/tasks/runners.yml

```diff
@@ -23,6 +29,11 @@
       Kind: "{{nodeLabel}}"
       Name: "e2e-{{nodeLabel}}-{{runId}}"
       ReaperMark: "e2e-testing-vm"
```

Comment:
Should I standardise the tag names with lowercase and _ separator?

Reply:
If that's the preference/standard, sounds good to me. Please remember to update the reaper pipeline.